_posts/2016-06-20-alpha-release-notes.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "argparse2cwl alpha release notes"
+date:   2016-06-20 02:27:27 +0300
+comments: true
+---
+
+`argparse2cwl` forms CWL tool descriptions from programs which use `argparse` as their argument parser. 
+
+## Prerequisites ##
+`argparse2cwl` is compatible with both Python 2 & 3.
+Dependencies:
+* Jinja2 >= 2.8
+
+## Installing ##
+
+Installation process:
+
+1. Clone the source repository:
+	
+	```$ git clone https://github.com/common-workflow-language/gxargparse.git ```  
+
+2. Change directory to the root folder of the project and switch to `cwl-refactor` branch:
+
+	```$ git checkout origin/cwl-refactor```
+
+3. Run 
+
+	```
+	   $ python setup.py install
+	   $ gxargparse_check_path 
+	```
+
+## Running ##
+
+To run `argparse2cwl`, call your tool as usually without actual arguments with `--generate_cwl_tool` option:
+
+	$ python cnvkit.py batch --generate_cwl_tool
```

Comment:
A reader might ask at this point:

1) Where do I get that `cnvkit.py` script from?
2) Does this mean that I have to modify my tools in oder to run `argparse2cwl`?

Providing a short explanation on those could help understanding better how pieces/scripts connect.